quinn-proto/src/shared.rs

```diff
@@ -240,6 +235,7 @@ impl Default for EndpointConfig {
 }
 
 impl EndpointConfig {
+    /// Validate if the local cid length is within the legal range.
```

Comment:
Logically, this validates the entire configuration; it just happens that there's currently only one field that we check.